src/core/experimental/buffer-map.js

```diff
@@ -0,0 +1,165 @@
+import GL from '../../webgl/gl-constants';
+import Buffer from '../../webgl/buffer';
+import TransformFeedback from '../../webgl/transform-feedback';
+import Model from '../model';
+import {isWebGL2, assertWebGL2Context} from '../../webgl/context';
+import assert from 'assert';
+
+const PASS_THROUGH_FS = `\
+void main()
+{
+}`;
+
+export default class BufferMap {
```

Comment:
I propose renaming to `Transform` as the official name for this class. Everyone talks about transform feedback and it relates to that, while sounding simpler (which is the purpose of the class).